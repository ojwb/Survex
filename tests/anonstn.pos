( Easting, Northing, Altitude )
(    3.00,     3.00,     4.00 ) 
(    3.00,     4.00,     3.00 ) 
(    4.00,     3.00,     3.00 ) 
(    0.00,    -0.00,    -5.00 ) 
(    0.00,    -0.00,    -5.00 ) 
(    0.00,    -0.00,    -5.00 ) 
(    0.00,    -0.00,    -5.00 ) 
(    0.00,    -0.00,   -10.00 ) 
(    0.00,     2.00,    -0.00 ) 
(    0.00,     0.00,     0.00 ) 1
(    3.00,     3.00,     3.00 ) 2
