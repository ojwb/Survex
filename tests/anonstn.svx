; pos=yes warn=0 error=0
*fix 1 0 0 0
1 . 2.00 000 0
1 ... 10 - down
*data normal tape compass clino from to
5.0 - down 1 .;comment
5.0 - down 1 ..;comment
5.0 - down 1 ...;comment
*alias station - ..
5.0 - down 1 -;comment
