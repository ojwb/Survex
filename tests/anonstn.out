
Removing trailing traverses...

Concatenating traverses...

Simplifying network...

Calculating network...

Calculating traverses...

Calculating trailing traverses...

Calculating statistics...

Survey contains 11 survey stations, joined by 10 legs.
There is 1 loop.
Survey has 2 connected components.
Total length of survey legs =   21.20m (  21.20m adjusted)
Total plan length of survey legs =    4.24m
Total vertical length of survey legs =   19.00m
Vertical range = 14.00m (from anonymous station at 4.00m to anonymous station at -10.00m)
Vertical range = 3.00m (from 2 at 3.00m to 1 at 0.00m)
North-South range = 4.00m (from anonymous station at 4.00m to 1 at 0.00m)
North-South range = 3.00m (from 2 at 3.00m to 1 at 0.00m)
East-West range = 4.00m (from anonymous station at 4.00m to 1 at 0.00m)
East-West range = 3.00m (from 2 at 3.00m to 1 at 0.00m)
   9 1-nodes.
   1 4-node.
   1 7-node.
